src/pages/resultsView/coExpression/CoExpressionVizUtils.ts

```diff
@@ -73,16 +81,16 @@ export function computePlotData(
         const yDatum = yData[uniqueSampleKey];
         if (xDatum && yDatum) {
             // only add data if data for both axes
-            const xVal = xDatum.value;
-            const yVal = yDatum.value;
+            const xVal = Number(xDatum.value);
+            const yVal = Number(yDatum.value);
             if (!isNaN(xVal) && !isNaN(yVal)) {
                 ret.push({
                     x: xVal,
                     y: yVal,
                     mutationsX: dispMut(xMutations[uniqueSampleKey] || []),
                     mutationsY: dispMut(yMutations[uniqueSampleKey] || []),
-                    profiledX: isProfiled(uniqueSampleKey, studyId, xHugoGeneSymbol, coverageInformation, studyToMutationMolecularProfile),
-                    profiledY: isProfiled(uniqueSampleKey, studyId, yHugoGeneSymbol, coverageInformation, studyToMutationMolecularProfile),
+                    profiledX: isProfiled(uniqueSampleKey, studyId, xGeneticEntityName, coverageInformation, studyToMutationMolecularProfile),
```

Comment:
coverageInformation only works for genes, not for genesets, so profiledX/Y should probably always be true if X/Y is a geneset, rather than false which is what this will give I'm pretty sure.